### ISSUE TITLE
CT Redesign Search: Exclude these school types: “For profit” filter has no label

### ISSUE THREAD

**Issue by delli-gatti-michael** [department-of-veterans-affairs/va.gov-team#29262]
The filters on the search results page of the redesigned Comparison Tool allow the user to exclude certain school types using a series of checkboxes. One of those types is "For profit". When tabbing to this checkbox, there is no "For profit" label so the screen reader only reads the checkbox group name, "Exclude these school types:".

### Steps to Reproduce:
1. With a screen reader running, navigate to https://staging.va.gov/gi-bill-comparison-tool-sandbox/
2. Execute any search that returns multiple results (ex. "college")
3. Tab to the "For profit" checkbox

![Screen Shot 2021-08-25 at 4 41 35 PM](https://user-images.githubusercontent.com/45575003/130861611-db6627a1-c246-451b-8142-b1d7d901e880.png)

### Expected result
The screen reader reads the name of the checkbox (all other checkboxes in this group function properly)

### Actual result
The screen reader only reads the checkbox group name

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/18537

**Comment by delli-gatti-michael** [comment]
Local testing complete

**Comment by delli-gatti-michael** [comment]
Testing complete
